src/win32_window.c

```diff
@@ -1462,6 +1462,41 @@ void _glfwPlatformDestroyWindow(_GLFWwindow* window)
         DestroyIcon(window->win32.smallIcon);
 }
 
+char* _glfwPlatformGetWindowTitle(_GLFWwindow* window)
+{
+    int count = GetWindowTextLengthW(window->win32.handle);
+    if(count == 0)
+    {
+        int error;
+        SetLastError(0);
```

Comment:
Shouldn't this be before the call to `GetWindowTextLengthW`?

Reply:
Yes, it should be that way...
I was probably taking the documentation too literally, even when it made no sense.